### PR TITLE
[KOA-5778] Set up Storybook for tokens

### PR DESCRIPTION
**Pull request by anambl**
Tested the config locally:
Screenshot 2023-01-23 at 17 00 44

Remember to include the following changes:

- [ ] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack-foundations/blob/main/CHANGELOG_FORMAT.md))
- [ ] `README.md`
- [ ] Tests
- [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +65,17 @@
     "@babel/preset-react": "^7.12.7",
     "@babel/register": "^7.12.1",
     "@skyscanner/stylelint-config-skyscanner": "^6.0.0",
-    "autoprefixer": "^10.4.12",
```
Comment (anambl) [inline package.json:66]: downgraded this (this is the same version we use on bpk web) as the latest version has some breaking changes which we did not account for when doing the upgrade as the config which uses this lib wasn't actually working

File: packages/bpk-stylesheets/webpack.config.babel.js [hunk 2]
```diff
@@ -21,7 +21,6 @@ import WrapperPlugin from 'wrapper-webpack-plugin';
 
 import postCssPlugins from '../../scripts/postCssPlugins';
 
-import babelConfig from './base.babel.config';
```
Comment (anambl) [inline packages/bpk-stylesheets/webpack.config.babel.js:24]: We were only using this for support for IE11 which we no longer support in Skyscanner

File: packages/bpk-stylesheets/base.js [hunk 3]
```diff
@@ -94,7 +94,7 @@
...
... System\n *\n * Copyright 2016 Skyscanner Ltd\n *\n * Licensed under the Apache License, Version 2.0 (the \"License\");\n * you may not use this file except in compliance with the License.\n * You may obtain a copy of the License at\n *\n *   http://www.apache.org/licenses/LICENSE-2.0\n *\n * Unless required by applicable law or agreed to in writing, software\n * distributed under the License is distributed on an \"AS IS\" BASIS,\n * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n * See the License for the specific language governing permissions and\n * limitations under the License.\n */\n\n\n(() => {\n  if (typeof document === 'undefined') {\n    return;\n  }\n  const classNames = [];\n\n  // touch support\n  classNames.push('ontouchstart' in window ||\n  // eslint-disable-line no-undef\n  window.DocumentTouch && document instanceof DocumentTouch // eslint-disable-line no-undef\n  ? 'touch-support' : 'no-touch-support');\n\n  // add more feature tests here...\n\n  // eslint-disable-next-line no-undef\n  document.documentElement.className += ` ${classNames.map(className => `bpk-${className}`).join(' ')}`;\n})();\n\n//# sourceURL=webpack:///./index.js?");
```
Comment (olliecurtis) [inline packages/bpk-stylesheets/base.js:97]: Assuming the change here is safe and doesn't cause any strange issues at all with some of the changes from `var` to `const`of the classNames array

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -141,6 +135,15 @@ You can also run the tests in 'watch mode', which means the process will continu
 
 </details>
 
+<details>
+<summary>Run the development environment</summary>
+
+We use [Storybook](https://storybook.js.org/) for our development environment. Run `npm storybook` to start the Storybook server, then go to [http://localhost:6006](http://localhost:6006) in a web browser to view it. We use Storybook to visualise any changes to our web tokens storing foundational design values such as spacing, colours etc.
```
[the remark this answers is not in the document]
Comment (anambl) [reply CONTRIBUTING.md:141]: ohhh good catch! yep, should be `npm run storybook`

File: packages/bpk-stylesheets/base.js (hunk 3 above)
Comment (anambl) [reply packages/bpk-stylesheets/base.js:97]: Yes, just double checked and both `const` and arrow functions are supported in the browsers versions that we support!

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**